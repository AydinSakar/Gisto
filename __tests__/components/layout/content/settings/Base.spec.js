import BaseSettings from 'components/layout/content/settings/Base';

const propSetup = (props) => ({
  ...props
});

const setup = (props) => mount(<BaseSettings { ...propSetup(props) }/>);

describe('COMPONENTS - <BaseSettings>', () => {
  test('render BaseSettings', () => {
    const component = setup();

    expect(component).toMatchSnapshot();
  });
});
