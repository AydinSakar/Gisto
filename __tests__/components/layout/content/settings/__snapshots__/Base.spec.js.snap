// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`COMPONENTS - <BaseSettings> render BaseSettings 1`] = `
<div>
  <h4>
    Color settings:
  </h4>
  <div>
    <Label>
      Base color:
      <InputColor
        color="#3F84A8"
        onChange={[Function]}
      />
    </Label>
    <Small>
      <strong>
        NOTE!
      </strong>
       colour change need app reload
    </Small>
  </div>
</div>
`;
